quupe-RN/android/app/src/main/java/com/quupe/MainApplication.java

```diff
@@ -26,9 +24,7 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
-            new RealmReactPackage(),
-            new VectorIconsPackage(),
-            new RNFirebasePackage()
+            new VectorIconsPackage()
```

Comment:
I think rebasing is not done properly because Realm is still part your project, please put it back through rebasing. Otherwise, really good work.